src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -29,50 +28,76 @@ object WdlExpression {
 
   val preEvaluableFunctions_SingleParameter: Seq[String] = Seq("read_int", "read_string")
 
+  // Function calls which can be called from within other function calls.
+  // e.g. read_int(stdout()) is fine.
+  val allowedInnerFunctionCalls = Seq[String]("stdout", "stderr")
+
   def functionCall(a: Ast): Boolean = a.getName == "FunctionCall"
 
   def binaryOperator(a: Ast): Boolean = binaryOperators.contains(a.getName)
   
-  def unaryOperator(a: Ast): Boolean = unaryOperators.contains((a.getName))
-  
+  def unaryOperator(a: Ast): Boolean = unaryOperators.contains(a.getName)
+
+  def getFunctionName(a: Ast): String = a.getAttribute("name").asInstanceOf[Terminal].getSourceString
```

Comment:
Also now that I see it used below I also think this would be better as a method on Ast (again, via implicit value class if that makes more sense)